commands/invite.mjs

```diff
@@ -0,0 +1,25 @@
+import { SlashCommandBuilder } from "@discordjs/builders";
+import { MessageEmbed } from 'discord.js';
+// import generalError from '../modules/general-error.mjs';
+
+const BOT_INVITE_LINK = "https://discord.com/api/oauth2/authorize?client_id=955521336904667227&permissions=309237664832&scope=bot%20applications.commands";
```

Comment:
I know it's been merged already but it looks like you can dynamically generate this:
[https://discord.js.org/#/docs/discord.js/stable/class/Client?scrollTo=generateInvite]()